soccer/MainWindow.cpp

```diff
@@ -718,14 +703,50 @@ void MainWindow::updateStatus() {
     RJ::Time curTime = RJ::now();
 
     // Determine if we are receiving packets from an external referee
-    bool haveExternalReferee =
-        (curTime - ps.lastRefereeTime) < RJ::Seconds(0.5);
+    bool haveExternalReferee = (curTime - ps.lastRefereeTime) < RJ::Seconds(1);
+
+    std::vector<int> validIds = _processor->state()->ourValidIds();
+
+    for (int i = 1; i <= Num_Shells; i++) {
+        if (std::find(validIds.begin(), validIds.end(), i - 1) !=
+            validIds.end()) {
+            // The list starts with None so i is 1 higher than the shell id
+            _ui.goalieID->setItemData(i, true, Qt::UserRole);
```

Comment:
Setting this to 
`setItemData(i, 33, Qt::UserRole - 1);` fixes this, but I want to try to find the proper way of disabling entries to a qcombobox.